src/Tribe/Views/V2/Widgets/Service_Provider.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * The main service provider for the version 2 of the Widgets.
+ *
+ * @since   TBD
```

Comment:
```suggestion
 * @since   TBD
 
```

Add a blank line before the `@package` entry.